File: 백준/Bronze/2675. 문자열 반복/문자열 반복.go
package main

import (
	"fmt"
)

func repeatString(char string, n int) string {
	var result string
	for i := 0; i < n; i++ {
		result += char
	}
	return result
}

func main() {
	var a, r int
	fmt.Scanln(&a)

	var s string

	for i := 0; i < a; i++ {
		fmt.Scanln(&r, &s)
		p := ""
		for _, char := range s {
			p += repeatString(string(char), r)
		}
		fmt.Println(p)
	}
}

File: 백준/Bronze/2675. 문자열 반복/문자열 반복.py
a = int(input())

result_list = []

for i in range(a):
    p = ""
    r, s = input().split()
    for j in range(len(s)):
        p = p + (s[j]*int(r))
    result_list.append(p)

for result in result_list:
    print(result)